alerts/jetty/README.md

```diff
@@ -0,0 +1,42 @@
+# Alerts for Jetty in the Ops Agent
+
+### Notification Channels
+For all alerts, a notification channel needs to be set up or the alert will fire silently.
+
+### User Labels
+User labels can be used for these policies by modifying the userLabels fields of the policies. i.e.
+
+```json
+{ 
+  "userLabels": {
+    "datacenter": "central"
+  }
+}
+```
+
+##  Access log failures
+
+If the amount of access log failures are above (defaulted to 10 `failed requests/min`) It could show your Server is having an issue serving pages.
+
+### Prerequisites
+For this alert, a new log based metric needs to be created.
+The name of the metric should be: 
+`jetty.request.failed.count`.
+
+The filter for the metric should be: 
+```
+logName:"jetty_access"
+httpRequest.status > 299
+```
+
+## Failed Config Parsing
+
+There are logs at WARN level at start up that Identify failed config parsing for modules.
```

Comment:
Should `Identify` be lowercase or is this a field?

Reply:
Whoops, updated.